src/main/java/uk/gov/cslearning/record/service/xapi/XApiService.java

```diff
@@ -75,30 +78,23 @@ public XApiService(XApiProperties xApiProperties) {
         return statements;
     }
 
-    public Collection<Statement> getStatements(String activityId) throws IOException {
```

Comment:
INFO - Is this method redundant?

Reply:
It's used [here](https://github.com/Civil-Service-Human-Resources/lpg-learner-record/blob/14a13fe37cdafd580f15772bc22802aa34af8d06/src/main/java/uk/gov/cslearning/record/service/UserRecordService.java#L69)

Reply:
A `getStatements` method that was deleted wasn't used anywhere.